src/lib/wallet/GoodWalletClass.js

```diff
@@ -1256,7 +1256,7 @@ export class GoodWallet {
     // we need around 400k gas to collect 1 bounty, so that's the minimum
     if (gas < 400000) {
       log.error('collectInvites low gas:', '', '', { gas, nativeBalance })
-      return false
+      throw new Error('There is not enough gas to collect bounty.')
```

Comment:
we probably have similar error messages elsewhere in any case please consult Pat and english native speaker when writing new texts

Reply:
@patpedrosa can you provide the correct copy to be displayed on the error modal

Reply:
now it redirects to outofgas screen as requested by @patpedrosa